Joesdev/dog-messenger
# FASP-73 Add Users ability to Opt Out of the Emails / Discontinue

## Whats This PR Do?
- Adds a function for unsubscribing users
- Adds a corresponding route
- Reformats the email to include a link to trigger the unsubscribe function

## Steps to Review
Run the following commands:
- `php artisan cache:clear`
- `php artisan migrate:refresh --seed`
- Enter the route `/test/notification/joesilvpb4@gmail.com`
- Open the gmail email and verify the format of the email is correct
- Click on the unsubscribe link and verify you are sent to the homepage with a message pertaining to being unsubscribed


Alternative Review Method if above doesn't work
- Review the ticket comments on trello, verify the unsubscription message is correct. (Screenshot of webpage)
